doc/developer_guide/psyir.rst

```diff
@@ -490,6 +490,42 @@ var2, var3)` would be represented by a
 The PSyIR supports the concept of named arguments for operation
 nodes, see the :ref:`named_arguments-label` section for more details.
 
+IntrinsicCall Nodes
```

Comment:
Thanks for adding the documentation, there is a bit about Symbols that also needs to be updated:
in the user_guide/psyir.rst:393 we list the Symbol subclasses, can you add InstrinsicSymbol (and the DataTypeSymbol which is missing).

Reply:
Done.